src/main/resources/fabric.mod.json

```diff
@@ -32,7 +32,8 @@
   "accessWidener": "blaze4d.aw",
   "depends": {
     "fabricloader": ">=0.11.3",
-    "minecraft": "1.17.x"
+    "minecraft": "1.17.x",
+    "fabric-language-kotlin": ">=1.6.2+kotlin.1.5.20"
```

Comment:
this is only needed for mods which use kotlin in the modInitialiser. we don't need it

Reply:
We have to include it since theres kotlin code here. It won't launch without kotlin libraries so I included it here.

Reply:
This should be handled from the rosella build which exports with needed libraries, not with this api.